dockerfile

```diff
@@ -18,6 +18,15 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     wget \
     && apt-get clean
 
+# Install security updates
+RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list.d/backports.list
+RUN apt-get update && apt-get -t bullseye-backports install -y \
+    libcurl3-gnutls=7.88.1-10+deb12u3~bpo11+1
+
+RUN echo "deb http://deb.debian.org/debian bookworm main" >> /etc/apt/sources.list.d/backports.list
```

Comment:
it is not ideal to rely on `bookworm` repository, but this zlib package is required by the system, the alternative is to `build from source` and manually replace the so* files, which IMO is even more hacky